vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.kt

```diff
@@ -801,6 +805,23 @@ class VectorSettingsPreferencesFragment : PreferenceFragmentCompat(), SharedPref
             false
         }
 
+        mUseDefaultStunPreference.let {
+            activity?.let { activity ->
+                it.isChecked = PreferencesManager.useDefaultTurnServer(activity)
+                val stun = activity.getString(R.string.default_stun_server)
+                it.summary = activity.getString(R.string.settings_call_ringtone_use_default_stun_sum, stun)
```

Comment:
no need to use `activity` to call `getString()`

Reply:
done